.compute

```diff
@@ -2,12 +2,14 @@
 
 set -xe
 
-apt-get install -y python3-venv
+apt-get install -y python3-venv libopus0
+
 python3 -m venv /tmp/venv
 source /tmp/venv/bin/activate
 
+pip install -U setuptools wheel pip
 pip install -r <(grep -v tensorflow requirements.txt)
-pip install tensorflow-gpu==1.15.0
+pip install tensorflow-gpu==1.14
```

Comment:
We're now on 1.15.0

Reply:
Because our cluster's CUDA version does not match.

Reply:
Where are the CUDA requirements for TF 1.15? I don't see them [here](https://www.tensorflow.org/install/source#linux).

Also, I know I've asked this before, but what CUDA version is our server at?

Reply:
FWIW, I've been using TF 1.15 on the cluster for a long time now. It works fine.

Reply:
Also, using an older version for training models after we upgraded the native client to use 1.15 can lead to unexpected behavior at the native client side. We should stick to using the same version on both sides.

Reply:
Could have been a problem with my own machine. Just started a test run to verify that 1.15 is actually working on the cluster.